src/coprocessor/codec/datum.rs

```diff
@@ -999,22 +1003,28 @@ pub fn split_datum(buf: &[u8], desc: bool) -> Result<(&[u8], &[u8])> {
     let pos = match buf[0] {
         INT_FLAG => number::I64_SIZE,
         UINT_FLAG => number::U64_SIZE,
-        BYTES_FLAG => bytes::encoded_bytes_len(&buf[1..], desc),
-        COMPACT_BYTES_FLAG => bytes::encoded_compact_len(&buf[1..]),
+        BYTES_FLAG => {
+            if desc {
```

Comment:
desc is never used. We can just remove it.

Reply:
done